src/Common/nunit/Logging/Logger.cs

```diff
@@ -53,10 +53,10 @@ public class Logger : ILogger
         /// <param name="writer">The writer where logs are sent.</param>
         public Logger(string name, InternalTraceLevel level, TextWriter writer)
         {
-            this.maxLevel = level;
+            maxLevel = level;
             this.writer = writer;
-            this.fullname = this.name = name;
-            int index = fullname.LastIndexOf('.');
+            fullname = this.name = name;
+            var index = fullname.LastIndexOf('.');
```

Comment:
The standard calls for judgement in the use of var. The author thought int made sense. I would not change that choice myself unless I were revising the entire file. Fair warning: I was the author here and I used int because (1) it's the same length as var and (2) it takes a moment thought to recall that the function returns int.